trustify/vulnerability-correlation-overview.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Understanding Vulnerability Correlation in Trustify
```

Comment:
You might consider changing this to be just "Vulnerability Correlation", but it's fine as-is